jetty-security/src/main/java/org/eclipse/jetty/security/authentication/LoginAuthenticator.java

```diff
@@ -34,7 +34,8 @@ public abstract class LoginAuthenticator implements Authenticator
 
     protected LoginService _loginService;
     protected IdentityService _identityService;
-    private boolean _renewSession;
+    private boolean _sessionRenew;
+    private int _sessionMaxInactiveInterval;
```

Comment:
Can we keep the naming consistent throughout? So `_renewSessionOnAuthentication`  and `_sessionMaxInactiveIntervalOnAuthentication`.